### ISSUE TITLE
Update JWT before expiration

### ISSUE THREAD

**Issue by OnionKiller** [kob07kob/durer-quiz-frontend#17]
FEATURE REQUEST (needs further discussion)
Add a feature to refresh JWT tokens just before it expires (5 mins). Need backen support rn.

**Comment by kob07kob** [comment]
Yeah perhaps increasing token time so it wont expire during the competition would  be an easier option.

**Comment by OnionKiller** [comment]
Yeah, it's more likely a future proofing issue. Afaik no backend atm.
